portal/guides/on-demand-minting.mdx

```diff
@@ -15,21 +15,32 @@ updated: ""
 
 ## Intro
 
-Our latest feature allows you to create unique ID's to hand out to anyone you want. Only users with that code will be able to mint or claim an unique NFT! thirdweb has created an easy component for you to integrate in your app! Let's take a look. Also you can find a GitHub repo [here](https://github.com/thirdweb-dev/examples/tree/main/typescript/signature-based-minting), where we show how to use this code.
+Our on-demand minting feature allows you to create unique ID's to hand out to anyone you want. Only users with that code will be able to mint or claim an unique NFT! thirdweb has created an easy component for you to integrate in your app! Let's take a look. Also you can find a GitHub repo [here](https://github.com/thirdweb-dev/examples/tree/main/typescript/signature-based-minting), where we show how to use this code.
```

Comment:
"create unique ID's" sounds weird to me, not sure what it means. the sentence after we say "only users with that code" so maybe just use "create unique codes" in the first sentence as well?

Reply:
fixed